templates/kotsadm-application.yaml

```diff
@@ -1,9 +0,0 @@
-{{- if .Values.kotsApplication }}
```

Comment:
Is this needed for branding? https://github.com/replicatedhq/embedded-cluster/blob/main/pkg/addons/adminconsole/adminconsole.go#L209-L221

Reply:
KOTS CLI will create it. Anything related to the app will be managed by KOTS/KOTS CLI.